adapters/unruly/unruly.go

```diff
@@ -113,7 +113,8 @@ func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest
 		}}
 	}
 
-	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
+	count := getBidCount(bidResp)
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(count)
```

Comment:
instead of computing count. could use  length of `internalRequest.imps`

Reply:
Sorry, missed this comment. Fixed.